tests/ci/cdk/cdk/codebuild/ec2_test_framework_omnibus.yaml

```diff
@@ -18,3 +18,17 @@ batch:
           EC2_AMI: "ami-0c29a2c5cf69b5a9c"
           EC2_INSTANCE_TYPE: "c6g.2xlarge"
           ECR_DOCKER_TAG: "amazonlinux-2023_clang-15x_sanitizer"
+          TARGET_TEST_SCRIPT: "./tests/ci/run_posix_sanitizers.sh"
+
+    - identifier: graviton2_tests_fips
```

Comment:
While we're here can you add a Graviton 4 instance as well?

Reply:
Added